### ISSUE TITLE
Invariant Violation: No callback found with cbID 1361 and callID 680 for RNSpotifyRemoteAuth.getSession

### ISSUE THREAD

**Issue by diegoareval** [cjam/react-native-spotify-remote#131]
when i open the intent to authenticate with spotify work very good, but when i click in cancel when i return to my app i got the next error:
Invariant Violation: No callback found with cbID 1361 and callID 680 for RNSpotifyRemoteAuth.getSession - most likely the callback was already invoked. Args: '[{"code":"SPTAuthorizationFailedErrorCode","message":"The operation couldn’t be completed. (com.spotify.sdk.login error 1.)","domain":"com.spotify.sdk.login","userInfo":{},"nativeStackIOS"

**Comment by diegoareval** [comment]
Exactly, i hit cancel and i get this issue, it's like there is no exist a handler when user cancel the auth flow

**Comment by cjam** [comment]
Sorry for the delay on this one, I've been pretty busy with other things, but hopefully I'll find some time to check it out.

**Comment by madhavisolanki** [comment]
any update on this. Someone was able to identify what is the issue?
